state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -932,8 +932,17 @@ impl<T: Config> Pallet<T> {
 			old_epoch,
 		);
 
-		let new_authorities = rotation_state.authority_candidates();
+		let mut new_authorities: Vec<<T as Chainflip>::ValidatorId> =
+			rotation_state.authority_candidates().into_iter().collect();
+		let hash = frame_system::Pallet::<T>::block_hash(
+			frame_system::Pallet::<T>::current_block_number(),
+		);
```

Comment:
Actually I don't think this will work - the block hash for the current block doesn't exist until *after* it's authored. 

We will have to use the parent hash (`frame_system::Pallet<T>::parent_hash()`).